src/kernel/arch/x86/interrupts.zig

```diff
@@ -40,7 +41,9 @@ export fn commonStub() callconv(.Naked) void {
         \\mov   %%esp, %%eax
         \\push  %%eax
         \\call  handler
-        \\pop   %%eax
+        \\add   $0x4, %%esp
```

Comment:
The add becomes redundant when you set esp on the next line

Reply:
Good point +1